launchable/commands/record/tests.py

```diff
@@ -26,17 +30,32 @@
     '--session',
     'session_id',
     help='Test session ID',
-    # validate session_id under debug mode,
-    required=os.getenv(REPORT_ERROR_KEY),
     type=str,
 )
+@click.option(
+    '--build',
+    'build_name',
+    help='build name',
+    type=str,
+    metavar='BUILD_NAME'
+)
 @click.pass_context
-def tests(context, base_path, session_id: str):
+def tests(context, base_path: str, session_id: str, build_name: str):
+    def create_session():
+        context.invoke(session, build_name=build_name, save_session_file=True)
+        return read_session(build_name)
+
     if not session_id:
-        click.echo(
-            "Session ID in --session is missing. It might be caused by Launchable API errors.", err=True)
-        # intentionally exiting with zero
-        return
+        if build_name:
+            try:
+                session_id = read_session(build_name)
+                if not session_id:
+                    session_id = create_session()
+            except SessionError as e:
+                session_id = create_session()
+        else:
+            raise click.UsageError(
+                'Missing option --build if you don\'t specify --session')
```

Comment:
This may be a matter of taste, but I think it'd be slightly better to say "Either --build or --session has to be specified". Curios to hear a native speaker's take, like @awilkes 

Here is my motivation. You get this error when you didn't specify either options. This error message as it's currently stated makes me as a user feels like the first decision that I need to make is if I should specify the --session option or not.